internal/worker/api/api.go

```diff
@@ -3,3 +3,5 @@
 package api
 
 const BasePath = "/api/worker/v1"
+const CloudBasePath = "/api/composer-worker/v1"
```

Comment:
I guess we should transition towards this being the only one, maybe expose both on the server side for now (in a follow-up)?

Reply:
I would actually prefer both paths to be implemented even in this PR. I think this makes this transition simpler and it's actually trivial: All we have to do is to call `api.RegisterHandlers` in server twice.